public/app/features/explore/ExplorePage.tsx

```diff
@@ -38,13 +37,11 @@ export default function ExplorePage(props: GrafanaRouteComponentProps<{}, Explor
   // if we were to update the URL on state change, the title would not match the URL.
   // Ultimately the URL is the single source of truth from which state is derived, the page title is not different
   useExplorePageTitle(props.queryParams);
-  const dispatch = useDispatch();
   const { keybindings, chrome } = useGrafana();
   const navModel = useNavModel('explore');
-  const [rightPaneWidthRatio, setRightPaneWidthRatio] = useState(0.5);
   const { width: windowWidth } = useWindowSize();
```

Comment:
this can also be moved inside `useSplitSizeUpdater` as we don't use its results anymore in this component